### PR TITLE
feat(uat): windows build python paho client

### PR DESCRIPTION
**Pull request by eschastlivtsev**
**Description of changes:**
-Update Python Paho client building system to make it possible running scenarios on Windows

**Why is this change necessary:**
Test all scenarios with Python Paho client on Windows

**How was this change tested:**
Run scenarios locally on Windows

**Test results:**
```
[INFO ] 2023-07-17 14:53:00.815 [main] StepTrackingReporting - Passed: 'GGMQ-1-T1-v3-paho-python: As a customer, I can connect, subscribe/publish at QoS 0 and 1 and receive using client application to MQTT topic'
[INFO ] 2023-07-17 14:53:00.815 [main] StepTrackingReporting - Passed: 'GGMQ-1-T1-v5-paho-python: As a customer, I can connect, subscribe/publish at QoS 0 and 1 and receive using client application to MQTT topic'
[INFO ] 2023-07-17 14:53:00.816 [main] StepTrackingReporting - Passed: 'GGMQ-1-T2-v3-paho-python: GGAD can publish to an MQTT topic at QoS 0 and QoS 1 based on CDA configuration'
[INFO ] 2023-07-17 14:53:00.816 [main] StepTrackingReporting - Passed: 'GGMQ-1-T2-v5-paho-python: GGAD can publish to an MQTT topic at QoS 0 and QoS 1 based on CDA configuration'
[INFO ] 2023-07-17 14:53:00.816 [main] StepTrackingReporting - Passed: 'GGMQ-1-T8-v3-paho-python: As a customer, I can configure local MQTT messages to be forwarded to a PubSub topic'
[INFO ] 2023-07-17 14:53:00.817 [main] StepTrackingReporting - Passed: 'GGMQ-1-T8-v5-paho-python: As a customer, I can configure local MQTT messages to be forwarded to a PubSub topic'
[INFO ] 2023-07-17 14:53:00.817 [main] StepTrackingReporting - Passed: 'GGMQ-1-T9-v3-paho-python: As a customer,I can configure local MQTT messages to be forwarded to an IoT Core MQTT topic'
[INFO ] 2023-07-17 14:53:00.817 [main] StepTrackingReporting - Passed: 'GGMQ-1-T9-v5-paho-python: As a customer,I can configure local MQTT messages to be forwarded to an IoT Core MQTT topic'
[INFO ] 2023-07-17 14:53:00.818 [main] StepTrackingReporting - Passed: 'GGMQ-1-T13-v3-paho-python: As a customer, I can connect two GGADs and send message from one GGAD to the other based on CDA configuration'
[INFO ] 2023-07-17 14:53:00.818 [main] StepTrackingReporting - Passed: 'GGMQ-1-T13-v5-paho-python: As a customer, I can connect two GGADs and send message from one GGAD to the other based on CDA configuration'
[INFO ] 2023-07-17 14:53:00.819 [main] StepTrackingReporting - Passed: 'GGMQ-1-T14-v3-paho-python: As a customer, I can configure IoT Core messages to be forwarded to local MQTT topic'
[INFO ] 2023-07-17 14:53:00.819 [main] StepTrackingReporting - Passed: 'GGMQ-1-T14-v5-paho-python: As a customer, I can configure IoT Core messages to be forwarded to local MQTT topic'
[INFO ] 2023-07-17 14:53:00.819 [main] StepTrackingReporting - Passed: 'GGMQ-1-T15-v3-paho-python: As a customer, I can configure Pubsub messages to be forwarded to local MQTT topic'
[INFO ] 2023-07-17 14:53:00.820 [main] StepTrackingReporting - Passed: 'GGMQ-1-T15-v5-paho-python: As a customer, I can configure Pubsub messages to be forwarded to local MQTT topic'
[INFO ] 2023-07-17 14:53:00.820 [main] StepTrackingReporting - Passed: 'GGMQ-1-T20-v3-paho-python: As a customer, I can associate and connect GGADs with GGC over custom port'
[INFO ] 2023-07-17 14:53:00.820 [main] StepTrackingReporting - Passed: 'GGMQ-1-T20-v5-paho-python: As a customer, I can associate and connect GGADs with GGC over custom port'
[INFO ] 2023-07-17 14:53:00.821 [main] StepTrackingReporting - Passed: 'GGMQ-1-T101-v3-paho-python: As a customer, I can use publish retain flag using MQTT V3.1.1'
[INFO ] 2023-07-17 14:53:00.821 [main] StepTrackingReporting - Passed: 'GGMQ-1-T102-v5-paho-python: As a customer, I can use new MQTT v5.0 features'
```

**Any additional information or context required to review the change:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml [hunk 1]
```diff
@@ -17,11 +17,10 @@ ComponentConfiguration:
     controlPort: 47619
 Manifests:
   - Artifacts:
-      - URI: classpath:/local-store/artifacts/client-python-paho
+      - URI: classpath:/local-store/artifacts/client-python-paho.exe
```
Comment (jcosentino11) [inline uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml:20]: can we have two manifests, one for linux, one for windows?

File: uat/testing-features/pom.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
             </activation>
             <properties>
                 <mosquitto.agent.artifact>mosquitto-test-client.amd64.tar.gz</mosquitto.agent.artifact>
-                <paho.python.agent.artifact>client-python-paho</paho.python.agent.artifact>
+                <paho.python.agent.artifact>client-python-paho.exe</paho.python.agent.artifact>
```
Comment (jcosentino11) [inline uat/testing-features/pom.xml:30]: is there a way to make it so linux doesn't have `.exe` extension?

File: uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml (hunk 1 above)
Comment (eschastlivtsev) [reply uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml:20]: At first, we created two manifests, but then we discovered that the OTF has a bug related to it.
It cannot parse platform-depending manifests at this moment.
Therefore, we decided to make the same name for both platforms to make it possible to run tests with this client on them.
When the bug will be fixed, we will update this recipe to have 2 manifests.

Comment (MikeDombo) [reply uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml:20]: Please fix the bugs

Comment (eschastlivtsev) [reply uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml:20]: Ok, this PR will wait for fix

Comment (bgklika) [reply uat/testing-features/src/main/resources/local-store/recipes/client_python_paho.yaml:20]: Maybe has no sense to wait for multi-platform support in OTF in that case due to we should customize artifact copying in maven. So will do some tricks in any case

**Pull request closed without merging**